addons/survey/wizard/survey_invite.py

```diff
@@ -209,8 +209,14 @@ def _get_answers_values(self):
 
     def _send_mail(self, answer):
         """ Create mail specific for recipient containing notably its access token """
-        subject = self.env['mail.render.mixin'].with_context(safe=True)._render_template(self.subject, 'survey.user_input', answer.ids, post_process=True)[answer.id]
-        body = self.env['mail.render.mixin']._render_template(self.body, 'survey.user_input', answer.ids, post_process=True)[answer.id]
+        subject_template = self.subject
+        body_template = self.body
+        if subject_template == self.template_id.subject and body_template == self.template_id.body_html and answer.partner_id.lang:
```

Comment:
I have tried this but there always are small differences between the sanitized template body (self.template_id.body_html) and the invite's body (see https://www.diffchecker.com/Z9jLlqUi). Is there a better way to compare html than `==` so that will not care about the order in which the style attributes are given ?

Reply:
checking that would probably be too complicated.

> I have tried this but there always are small differences between the sanitized template body (self.template_id.body_html) and the invite's body (see https://www.diffchecker.com/Z9jLlqUi).

then it seems to be done by the editor/browser rendering, but if it's done by the editor/browser rendering I would not have thought that adding sanitize=False would fix anything, does the current solution work?

Reply:
Yes it works

Reply:
I have tried, and I see the same diff with or without the `sanitize=False`, the content is also more different than you (probably that some of the modification comes from the editor, and you saved the editor one time on the mail.template so part of the difference was removed at that point)

https://www.diffchecker.com/QtSx2RtN